images/linux/scripts/SoftwareReport/SoftwareReport.Browsers.psm1

```diff
@@ -23,6 +23,12 @@ function Get-ChromiumVersion {
     return $chromiumVersion
 }
 
+function Get-EdgeVersion {
+    $edgeVersion = microsoft-edge --version
+    $aptSourceRepo = Get-AptSourceRepository -PackageName "microsoft-edge"
+    return "$edgeVersion (apt source repository: $aptSourceRepo)"
```

Comment:
looks like we don't need whitespace here
![image](https://user-images.githubusercontent.com/48208649/177790458-df7ee7b9-e88b-4704-bdda-fc44bc06c4e1.png)

Reply:
trimmed it, lets go again